features/admin.policy-administration.v1/components/policy-administration-page-layout.tsx

```diff
@@ -0,0 +1,374 @@
+                                    { isAlgorithmLoading && <CircularProgress size={ 12 } /> }
+                                </Stack>
+                            </Button>
+                        </Grid>
+                        <Grid xs={ 6 }>
+                            <PrimaryButton
+                                onClick={ () => setShowWizard(true) }
+                            >
+                                <Icon name="add" />
+                                { t("policyAdministration:buttons.newPolicy") }
+                            </PrimaryButton>
+                        </Grid>
+                    </Grid>
+                )
+            }
+            data-componentid={ componentId }
+            className="policy-administration-page"
+        >
+
+
+            <Grid container>
+                <Grid xs={ 12 } sm={ 12 } md={ 12 } lg={ 12 } xl={ 12 }>
+                    <AdvancedSearchWithBasicFilters
+                        fill="white"
+                        onFilter={ handleListFilter }
+                        filterAttributeOptions={ [
+                            {
+                                key: 0,
+                                text: t("tenants:listing.advancedSearch.form.dropdown.filterAttributeOptions.domain"),
+                                value: "domainName"
+                            }
+                        ] }
+
+                        placeholder={ t("policyAdministration:advancedSearch.placeholder") }
+                        defaultSearchAttribute={ "policyName" }
+                        defaultSearchOperator="co"
+                    />
+                </Grid>
+            </Grid>
+
+            <Grid container spacing={ 2 } marginTop={ 2 } >
+                <Grid xs={ 6 }>
+                    <DnDProvider>
+                        <Typography variant="h5" className="policy-list-header">Active Policies</Typography>
+                        <Card id={ "active-policy-list-container" } className="policy-list-card">
+                            <InfiniteScroll
+                                next={ fetchMoreActivePolicies }
+                                hasMore={ hasMoreActivePolicies }
+                                loader={
+                                    (<div style={ { textAlign: "center" } }>
+                                        <CircularProgress />
+                                    </div>)
+                                }
+                                dataLength={ activePolicies.length }
+                                scrollableTarget={ "active-policy-list-container" }
+                                style={ { overflow: "unset" } }
+                            >
+                                <CardContent>
+                                    <PolicyList
+                                        containerId="1"
+                                        policies={ activePolicies } // Use your active policies array
```

Comment:
Let's remove unnecessary comments.

```suggestion
                                        policies={ activePolicies }
```